public/data/cambodia/idpoor-back.json

```diff
@@ -0,0 +1,6312 @@
+{
```

Comment:
I'm assuming this file is now redundant as we fetch it from the prebuild script? Can we name it `idpoor-backup.json` and add some documentation in prebuild.ts to explain what this file is?

Reply:
I agree. I am working on a new approach which is going to better the process